tools/erc-repository-indexer/erc-contract-indexer/tests/acceptance/acceptance.test.ts

```diff
@@ -0,0 +1,116 @@
+/*-
+ *
+ * Hedera Smart Contracts
+ *
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+import dotenv from 'dotenv';
+dotenv.config();
+import testHelper from './utils/helper';
+import testConstants from './utils/constants';
+import { ercRegistryRunner } from '../../src/runner';
+import { Helper } from '../../src/utils/helper';
+import constants from '../../src/utils/constants';
+import NodeClient from '@hashgraph/sdk/lib/client/NodeClient';
+
+describe('ERC Registry Acceptance Test', () => {
+  // production networks take more time to finish deployments
+  jest.setTimeout(60000);
```

Comment:
nit: i thought there used to be a way to test/poll that local node was ready. 
@natanasow are you aware of any reliable way?
Nothing to do here but something to think about

Reply:
Gotcha thanks for bringing it up!